### DISCUSSION TITLE
Charts bars data is sometimes blurry on resize

### DISCUSSION THREAD

**Discussion by dtau00** [tradingview/lightweight-charts#1544] [Q&A]
i'm using a candlestick bars chart.  Sometimes when i resize the charts, when im programatically changing the size of the charts, the bars are blurry (not pixel perfect).  I notice when im manually resizing the chart (via css resize), the chart sometimes gets jumpy, as if it's resizing itself for pixel perfect rendering.  

My question is, is there a way to manually invoke the function to re-render the chart bars so it is pixel perfect?

**Comment by dss010101** [comment]
same here. any solution to this?
